tardis/visualization/tests/test_plot_util.py

```diff
@@ -125,7 +126,7 @@ def test_extract_and_process_packet_data(
 
         expected_df = pd.DataFrame(expected_data)
 
-        line_mask = (expected_df["last_interaction_type"] > -1) & (
+        line_mask = (expected_df["last_interaction_type"] > InteractionType.NO_INTERACTION) & (
             expected_df["last_line_interaction_in_id"] > -1
```

Comment:
perhaps this `-1` can also use `InteractionType.NO_INTERACTION`

Reply:
Fixed in https://github.com/tardis-sn/tardis/pull/3276/commits/7664639f61e9b863a916466b9dec1f484c2c5d34